cypress/support/commands.js

```diff
@@ -23,3 +23,10 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+import 'cypress-file-upload';
+import '@testing-library/cypress/add-commands';
+
+Cypress.Commands.add('sqlFixture',(sqlFilename)=>{
+    cy.exec(`docker exec dm_db psql -U digitalmarketplace digitalmarketplace -f /workspace/cypress/fixtures/${sqlFilename}`) //need /workspace/ because that's the file structure in the db container
+})
```

Comment:
This based on how [CAS does it-](https://github.com/bcgov/cas-cif/pull/194/files)-couldn't get theirs to work so tweaked, hopefully this is cool too